docs/dev/settings_files.rst

```diff
@@ -4,37 +4,42 @@
 Settings Files
 ===============================================================================
 
-Several of the scripts provided as part of PUDL require more arguments than can
-be easily managed on the command line, and it's useful to preserve a record of
-how the data processing pipeline was run, so they read their settings from YAML
-files, examples of which are included in the distribution.
+Several of the scripts provided as part of PUDL require more arguments than can be
+easily managed on the command line, and it's useful to preserve a record of how the
+data processing pipeline was run, so they read their settings from YAML files,
+examples of which are included in the distribution.
```

Comment:
I would split this into several sentences for clarity.

Reply:
split.